components/avatar/ShapeContext.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+export type AvatarShape = 'circle' | 'square';
+
+const ShapeContext = React.createContext<AvatarShape>('circle');
+
+export interface ShapeContextProps {
+  shape?: AvatarShape;
```

Comment:
I would suggest using the name `maxShape` as this refers to the max container of the group component in line with `maxPopoverTrigger` and `maxCount` etc.